tradeit-android-sdk/src/main/java/it/trade/android/sdk/model/TradeItLinkedBrokerParcelable.java

```diff
@@ -255,6 +255,16 @@ public boolean equals(Object o) {
         return linkedLogin.userId.equals(that.linkedLogin.userId);
     }
 
+    public boolean equalsAccounts(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+
+        TradeItLinkedBrokerParcelable that = (TradeItLinkedBrokerParcelable) o;
+
+        return linkedLogin.userId.equals(that.linkedLogin.userId) &&
```

Comment:
Do we want to update only when there is an update on accounts or also other properties (label, userToken, broker, accountLastUpdated, error) ?